IdentityCore/src/MSIDOAuth2Constants.m

```diff
@@ -96,7 +96,7 @@
 NSString *const MSID_SPE_INFO_CACHE_KEY                  = @"spe_info";
 NSString *const MSID_RESOURCE_RT_CACHE_KEY               = @"resource_refresh_token";
 NSString *const MSID_ACCOUNT_ID_CACHE_KEY                = @"authority_account_id";
-NSString *const MSID_ACCOUNT_TYPE_CACHE_KEY              = @"account_type";
```

Comment:
Any reason why we don't change `MSID_ACCOUNT_TYPE_CACHE_KEY` to `MSID_AUTHORITY_TYPE_CACHE_KEY`?

Reply:
Done